src/main/kotlin/carracing/CarRace.kt

```diff
@@ -1,7 +1,8 @@
 package carracing
 
 import carracing.RandomNumberGenerator.getRandom
-import carracing.view.ResultView
+
+typealias RaceHistory = List<List<Car>>
```

Comment:
Continued with https://github.com/next-step/kotlin-racingcar/pull/1813#discussion_r2041653288

> So if I understand correct, the RaceHistory will be List<List<Car>>, right?

Yes, That's right! 

Just a thought: what would change if RaceHistory were its own class rather than a typealias?
Would it give you more flexibility in handling race data or expressing domain logic more clearly?